lib/iris/fileformats/grib/_grib_cf_map.py

```diff
@@ -119,6 +121,7 @@
     G2Param(2, 0, 7, 8): CFName(None, 'storm_relative_helicity', 'J kg-1'),
     G2Param(2, 0, 14, 0): CFName('atmosphere_mole_content_of_ozone', None, 'Dobson'),
     G2Param(2, 0, 19, 1): CFName(None, 'grib_physical_atmosphere_albedo', '%'),
+    G2Param(2, 2, 0, 0): CFName('land_binary_mask', None, '1'),
```

Comment:
`land_binary_mask` appears to have the same `discipline`, `category` and `number` as `land_area_fraction` below.